packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -75,6 +79,26 @@ function itemFilter<T extends string | number>(inputValue: string) {
 const noop = () => {};
 const asyncNoop = () => Promise.resolve([]);
 
+function logOptions(
```

Comment:
this function should be removed in favor of the new from from utils yeah?

Reply:
Yes! I ran into some betterer issues and wasn't sure if I had added them so I only pushed part of the commit, just the new files. It is removed!